client/src/Clusters/TopCharts/mockData.js

```diff
@@ -0,0 +1,158 @@
+import emptyAppImage from './Images/emptyAppImage.png';
+import facebookLogo from './Images/facebookLogo.svg';
+import pandoraLogo from './Images/pandoraLogo.png';
+import myWeatherLogo from './Images/myWeatherLogo.png';
+import wearOSLogo from './Images/wearOSLogo.png';
+
+export const topFree = { 
+      title: "Top Free",
+      cards: [
+      {
+        uniqueID: "01",
```

Comment:
i would just name this "id" or "appId", which is closer to what the data from the backend will be